packages/api/src/hooks/useActiveWalletAccountVerbose.ts

```diff
@@ -0,0 +1,54 @@
+import { useMemo } from 'react';
+import useActiveWalletAccount from './useActiveWalletAccount';
+import useMT5AccountsList from './useMT5AccountsList';
+import useWalletAccountsList from './useWalletAccountsList';
+import useDxtradeAccountsList from './useDxtradeAccountsList';
+
+/** A custom hook that returns the wallet object for the current active wallet, containing `transfer_options` list. */
+const useActiveWalletAccountVerbose = () => {
+    const { data: wallet, isError: isWalletError, isLoading: isWalletLoading, ...rest } = useActiveWalletAccount();
+    const { data: walletList, isError: isWalletListError, isLoading: isWalletListLoading } = useWalletAccountsList();
+    const {
+        data: mt5List,
+        isError: isMT5AccountsListError,
+        isLoading: isMT5AccountsListLoading,
+    } = useMT5AccountsList();
+    const { data: dxtradeList, isError: isDxtradeError, isLoading: isDxtradeLoading } = useDxtradeAccountsList();
+
+    const dtrade = useMemo(
+        () => wallet?.linked_to?.filter(account => account.platform === 'dtrade'),
+        [wallet?.linked_to]
+    );
+
+    const dxtrade = useMemo(
+        () =>
+            wallet?.linked_to
+                ?.filter(account => account.platform === 'dxtrade')
+                .map(account => ({ ...account, ...dxtradeList?.find(acc => acc.login === account.loginid) })),
+        [dxtradeList, wallet?.linked_to]
+    );
+
+    const mt5 = useMemo(
+        () =>
+            wallet?.linked_to
+                ?.filter(account => account.platform === 'mt5')
+                .map(account => ({
+                    ...account,
+                    ...mt5List?.find(acc => acc.loginid === account.loginid),
+                })),
+        [mt5List, wallet?.linked_to]
+    );
+
+    return {
+        /** User's current active wallet. */
+        data: {
+            ...wallet,
+            transfer_options: { dtrade, dxtrade, mt5, wallets: walletList },
```

Comment:
hey @rostislav-deriv 
actually i'm confused about `transfer_options` field name and the whole idea of the hook itself. IMHO there is no relation between what we are doing in the hook and return value of the hook. Since you are trying to get all info about linked accounts and just pass them as `transfer_options` property make other devs confused about real usage of the hook (getting accounts linked to particular wallet).

Reply:
basically the same thing that @thisyahlen-deriv mentioned above
got it that we will rename it!

Reply:
The hook is no more